modules/bitgo/test/v2/unit/wallet.ts

```diff
@@ -3743,4 +3743,108 @@ describe('V2 Wallet:', function () {
     });
   });
+  describe('NFT Tests', function () {
+    let ethWallet: Wallet;
+
+    before(async function () {
+      const walletData = {
+        id: '598f606cd8fc24710d2ebadb1d9459bb',
+        coin: 'gteth',
+        keys: [
+          '598f606cd8fc24710d2ebad89dce86c2',
+          '598f606cc8e43aef09fcb785221d9dd2',
+          '5935d59cf660764331bafcade1855fd7',
+        ],
+        multisigType: 'onchain',
+      };
+      ethWallet = new Wallet(bitgo, bitgo.coin('gteth'), walletData);
+    });
+
+    afterEach(async function () {
+      nock.cleanAll();
+    });
+
+    it('Should return all nfts in the wallet', async function () {
+      const getTokenBalanceNock = nock(bgUrl)
+        .get(`/api/v2/gteth/wallet/${ethWallet.id()}?allTokens=true`)
+        .reply(200, {
+          ...walletData,
+          nfts: {
+            '0x8397b091514c1f7bebb9dea6ac267ea23b570605': {
+              type: 'ERC721',
+              metadata: {
+                name: 'terc721:bitgoerc721',
+                tokenContractAddress: '0x8397b091514c1f7bebb9dea6ac267ea23b570605',
+              },
+              collections: {},
+              balanceString: '0',
+              confirmedBalanceString: '0',
+              spendableBalanceString: '0',
+              transferCount: 0,
+            },
+            '0x87cd6a40640befdd96e563b788a6b1fb3e07a186': {
+              type: 'ERC1155',
+              metadata: {
+                name: 'terc1155:bitgoerc1155',
+                tokenContractAddress: '0x87cd6a40640befdd96e563b788a6b1fb3e07a186',
+              },
+              collections: {},
+              balanceString: '0',
+              confirmedBalanceString: '0',
+              spendableBalanceString: '0',
+              transferCount: 0,
+            },
+            '0xtnonstandard:token': {
+              type: 'NONSTANDARD',
+              metadata: {
+                name: 'tnonstandard:token',
+                tokenContractAddress: '0xtnonstandard:token',
+              },
+              collections: {},
+              balanceString: '0',
+              confirmedBalanceString: '0',
+              spendableBalanceString: '0',
+              transferCount: 0,
+            },
+            '0xterc721:token': {
+              type: 'ERC721',
+              metadata: { name: 'terc721:token', tokenContractAddress: '0xterc721:token' },
+              collections: {},
+              balanceString: '0',
+              confirmedBalanceString: '0',
+              spendableBalanceString: '0',
+              transferCount: 6,
+            },
+            '0xterc1155:token': {
+              type: 'ERC1155',
+              metadata: {
+                name: 'terc1155:token',
+                tokenContractAddress: '0xterc1155:token',
+              },
+              collections: {},
+              balanceString: '0',
+              confirmedBalanceString: '0',
+              spendableBalanceString: '0',
+              transferCount: 0,
+            },
```

Comment:
Would be nice to define this mocked api-response somewhere (a json fixture) and import it here. Makes it re-usable.